test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -1184,4 +1185,35 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 			await driverHelper.acceptAlertIfPresent( driver );
 		} );
 	} );
+
+	describe( 'Can Share Posts From Reader (PressThis)! @parallel', function() {
```

Comment:
Does it make more sense to have all of this in the Reader spec (https://github.com/Automattic/wp-calypso/blob/master/test/e2e/specs/wp-reader-spec.js)?

Reply:
It'd mainly be an editor breakage, if the loading order changes again, we'll end up with a title set but no post content. I'm also using an experimental function to make this work the last time it regressed.